salt/api-gateway/config/lib-systemd-system-kong.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description="kong, api-gateway"
+After=network.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+StandardError=syslog
```

Comment:
is it possible to put the stderr in a file somewhere? I don't think Kong would produce much on stderr being a daemon, but maybe in an existing /usr/local/kong/logs (can't remember the exact path) file?